scaper/core.py

```diff
@@ -1721,7 +1723,8 @@ def _generate_audio(self, audio_path, ann, reverb=None,
                         cmb.convert(samplerate=self.sr,
                                     n_channels=self.n_channels,
                                     bitdepth=None)
-                        # Then trim
+                        # Then trim, using the duration as the lower limit to handle
```

Comment:
what duration? source duration? event duration? soundscape duration? Need to be specific ;)